File: Micro Py/indexPub.py
import paho.mqtt.client as mqtt
import time,random
i = 0
mqttClient = mqtt.Client()
mqttClient.connect("192.168.56.1",1883)
mqttClient.loop_start()
while True:
    # data = "success"
    # data += str(i)
    data = str(random.randint(0,9))
    mqttClient.publish('test', data)
    print(data,i)
    i = i + 1
    time.sleep(3)

File: Micro Py/indexPub.js
var mqtt = require('mqtt');
var client = mqtt.connect({ host: '192.168.50.248', port: 1883 });

client.on('connect',function(){
  console.log("publish connect broker");
});
// client.on('connect',function(){
//   console.log("connecting")
//   client.subscribe("test1")
//   client.subscribe("second")
// })
// client.on('message',function(topic,message){
//    console.log(topic,message.toString())
// })
var i=0;
setInterval(function(){
  /*var text='successful: ';
  
  text+=i.toString();
  console.log("test: ",text );
  client.publish('test', text);*/
  var pubb = i % 10;
  var pub = pubb.toString();
  client.publish('test', pub);
  console.log(pub);
  i++;
}, 5000);



